pages/governance.html

```diff
@@ -10,193 +10,106 @@
 <div class="container-text">
 	<h3>Legal status</h3>
 	<p>
-		On its own, Godot has no legal status. Godot exists as a member project of
-		the <a href="https://sfconservancy.org">Software Freedom Conservancy</a>
-		(SFC). The SFC provides a home and infrastructure for Free, Libre, and Open
-		Source Software (FLOSS) projects by aggregating the work necessary for
-		running a non-profit organization including accounting and legal compliance.
-		For more information about what the SFC provides, please refer to <a
-			href="https://sfconservancy.org/projects/services/">their website</a>.
+		The Godot Engine is financially supported by the <a href="https://godot.foundation">Godot Foundation</a>, a non-profit organization formed on August 23rd, 2022 via the KVK (number 87351919). The Godot Foundation is responsible for managing donations made to the Godot Project and ensuring that such donations are used to enhance the Godot Engine. The Godot Foundation is a legally independent organization and does not own the Godot Project. In the past, the Godot Project existed as a member project of the <a href="https://sfconservancy.org"><strong>Software Freedom Conservancy</strong></a>.
```

Comment:
> `...2022 via the KVK (number 87351919)`

I would add "In the Netherlands" as most people won't be able to place what KVK means.